### DISCUSSION TITLE
Can we add our own classes instead of gjs in GrapeJS in built in blocks or elements?

### DISCUSSION THREAD

**Discussion by ZaheerNiazi007** [GrapesJS/grapesjs#5879] [Q&A]
I know about this plugin [https://github.com/silexlabs/grapesjs-ui-suggest-classes](https://chatgpt.com/c/url), but what I want is that when I drag a block onto the canvas, instead of show 'gs-frames' class by default, I will show according to my wish name like op-frames .
is it possible?
![image](https://github.com/GrapesJS/grapesjs/assets/72734684/fdfcb399-fe1a-4d08-939a-92666402ad56)

**Comment by lexoyo** [comment]
Hello
The gjs-* classes are used only on the editor's elements, not in your website elements. Your website is in this iframe .gjs-frame

The plugin shows the classes your users will add to the website

I guess I don't understand your question
